src/scss/site/_dashboard.scss

```diff
@@ -5,62 +5,62 @@
 .dashboard {
   input[name*="ACCESS1-0"] + .bx--checkbox-label::before,
   input.bx--checkbox[name*="ACCESS1-0"]:checked + .bx--checkbox-label::before {
-    background-color: --gcm-ACCESS1-0;
-    border-color: --gcm-ACCESS1-0;
+    background-color: var(--gcm-ACCESS1-0);
```

Comment:
Not a big deal to use the CSS variables but couldn't you use the Sass equivalent of these variables here?

Reply:
Fixed via https://github.com/berkeley-gif/caladapt-website-2021/pull/37/commits/32af8060f52a331204ef8d1cee5d519a0aea1f22